src/cdk/collections/selection.spec.ts

```diff
@@ -279,4 +279,24 @@ describe('SelectionModel', () => {
     let singleSelectionModel = new SelectionModel();
     expect(singleSelectionModel.isMultipleSelection()).toBe(false);
   });
+
+  it('should deselect value if comparable to another one', () => {
+    type T = {key: number; value: string};
```

Comment:
Instead of `T`, this should use a proper name like `Item`. We use `T` to name the generic params.

Reply:
Agree, corrected!